click/simplelog.py

```diff
@@ -0,0 +1,199 @@
+from .core import Option
+import logging
+
+
+def simplelog(*args, **kwargs):
+
+    """ Decorator to add a very simple logging facility to scripts
+
+    For parameters see SimpleLog.__init__
+    """
+
+    def make_simplelog(f):
+
+        return SimpleLog(f, **kwargs)
+
+    return make_simplelog
+
+
+class SimpleLog:
+
+    __name__ = "SimpleLog"
+
+    def __init__(self, callback, verbose_help=None, debug_help=None,
```

Comment:
That's massive. No idea how to fix this, but I think this is not really acceptable. Maybe users should override methods, which gives them more flexibility anyway?